frontend/README.md

```diff
@@ -1,15 +1,38 @@
+# PIMS Frontend Application - React
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-# Setup
+## Frontend Environment Variables
 
 Create a .env file and populate it with this.
 
-```
+```conf
 NODE_ENV=development
-KEYCLOAK_URL=http://host.docker.internal:8080/auth/admin/realms/quartech/users
 CHOKIDAR_USEPOLLING=true
 ```
```

Comment:
Please add **API_URL** here (as you removed it from `docker-compose.yaml` before